__tests__/app.test.js

```diff
@@ -138,21 +138,55 @@ describe("/api/articles", () => {
                 });         
         });
     });
-    test("GET: 200 - articles are ordered by created_at in descending order", ()=>{
+    test("GET: 200 - articles are ordered by created_at in descending order by default", ()=>{
         return request(app)
         .get("/api/articles?sort_by=created_at")
         .expect(200)
         .then(({body}) => {
             expect(body.articles).toBeSortedBy("created_at", { descending: true})
         })
     });
-    test('GET: 200 - articles can be ordered ascending', () => {
+    test("GET: 200 - articles can be ordered ascending", () => {
         return request(app)
           .get('/api/articles?order=asc')
           .expect(200)
           .then(({ body }) => {
             expect(body.articles).toBeSortedBy('created_at');
-          });
+        });
+    });
+    test("GET: 200 - orders by default if sort_by is misspelt", () => {
+        return request(app)
+          .get('/api/articles?soort_by=title')
```

Comment:
I think this description could be phrased better. It's not the case that we're testing explicitly for sort_by being misspelt - it's that we're checking for a query that doesn't exist. This could be `soort_by=title` or it could be `manj=title`, it would still require the same response.

Reply:
In addition - what if the sort_by _value_ was something that is invalid. For example `sort_by=manj`